r/BANEScarparkinglite/R/web_scraping.R

```diff
@@ -11,31 +11,34 @@
 #' @param x A vector of years of seasons to retrieve records from.
 #' @return A data frame of kick-off date-times and match outcomes.
 #' @examples
+#' # Return matches from 2016/17 season
+#' rugby <- get_rugby(2017)
+#' \donttest{
 #' # Return matches from 2014/15, 2015/16 seasons
 #' seasons <- c(2015, 2016)
 #' rugby <- get_rugby(seasons)
-#'
+#' }
 #' @export
 
 get_rugby <- function(x) {
-
+    
     # Set up data frame, to be added to shortly
     rugby <- data.frame(GMT = character(0), HomeWin = logical(0))
-
+    
```

Comment:
this added whitespace, is that an R style convention? I've only ever worked with code where whitespace-only blank lines are frowned upon

Reply:
There's no real standard here - though it doesn't matter hugely (whitespace is not important in R), I try to be consistent with indentation. Interesting question for the equivalent in Python [here](
https://stackoverflow.com/questions/2727988/python-indentation-in-empty-lines).

Also, if you ask RStudio to "reindent" or "reflow" your code, it **will** indent empty lines:

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/21007837/35671643-fa013a6e-0733-11e8-9a17-04591fc4ca04.gif)

Reply:
Ew